src/smart-components/group/add-group/users-list.js

```diff
@@ -78,6 +97,11 @@ const UsersList = ({ users, fetchUsers, isLoading, pagination, selectedUsers, se
     });
   };
 
+  const updateFilters = (payload) => {
+    inModal || updateUsersFilters(payload);
+    setFilters(payload);
```

Comment:
Question. Do we keep the same state in two places? Can we use only one state storage? (it's fine if not) We wouldn't have to synchronize it.

Reply:
Removed usage of `useState`, it's actually not needed. This function is called to update filters in redux before data fetching actually succeeds. Otherwise, filter tags don't disappear immediately after clearing the filters but just after fetching the data.

Reply:
If we want to have an immediate visual response, we can set the data within the `_PENDING` action and roll them back inf the request fails in the `_REJECTED` action. We can do it in a different PR.